services/storage/docker-compose.yml

```diff
@@ -23,8 +23,10 @@ services:
       - ../nats/client-cert.pem:/etc/neofs-node/nats.tls.cert
       - ../nats/client-key.pem:/etc/neofs-node/nats.tls.key
       - ../nats/ca-cert.pem:/etc/neofs-node/nats.ca.crt
+      - ./cfg:/etc/neofs-node/cfg
     stop_signal: SIGKILL
-    env_file: [ ".env", ".storage.env", ".int_test.env" ]
```

Comment:
Maybe we should keep `.storage.env` to simplify overriding?

Reply:
does your suggestion include keeping the empty `.storage.env` in the repo?